src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostExprTyping.scala

```diff
@@ -144,6 +144,18 @@ trait GhostExprTyping extends BaseTyping { this: TypeInfoImpl =>
         }
       }
     }
+
+    case expr: PPermission => expr match {
+      case PFullPerm() => noMessages
+      case PNoPerm() => noMessages
+      case PFractionalPerm(left, right) => (intConstantEval(left), intConstantEval(right)) match {
+        case (l, r) if l.isEmpty || r.isEmpty => error(left, s"expected a constant, but got $left", l.isEmpty) ++
+          error(right, s"expected a constant, but got $right", r.isEmpty)
+        case (_, Some(dividend)) => error(right, s"expected a non-zero dividend, but got $right", dividend == 0)
```

Comment:
I actually tried that but Scala is for some reason not able to implicitly convert `0` to a `BigInt`. Because `BigInt` also does not seem to have an unapply function, the best imho I can do is this: `case (_, Some(zero)) if zero == 0 => ...`